QSG 2025 changes

RyanMocke opened the pull request:
Updated the quickstart guides to be more compact. Added images, links, formatting.

Mocke44 committed:
QSG 2025 changes

Updated the quickstart guides to be more compact. Added images, links, formatting.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/mendix/docs?pullRequest=9926) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/mendix/docs?pullRequest=9926) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: ConnorLand<br/>:x: Mocke44<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/mendix/docs?pullRequest=9926) it.</sub>

Mocke44 committed:
Renamed QSG Part 1 and 2.

Incorrectly changed the file names in original PR. Reverted file names to use existing names so as to not affect navigation.

ConnorLand committed:
Merge branch 'development' into QSG2025

jdiehl commented on content/en/docs/quickstarts/native-mobile-app.md:
```diff
@@ -1,205 +1,176 @@
+The Mendix Platform enables you to build apps of many different kinds, including web, native mobile, and PWA. When choosing to build native mobile applications, there are some special requirements to keep in mind:
 
-## Prerequisites
+* [Offline-first development](https://docs.mendix.com/refguide/mobile/building-efficient-mobile-apps/offlinefirst-data/): Native mobile apps run on devices that cannot guarantee a stable connection to the internet all the time. When developing these types of apps, it's important to design your app with this in mind. Data is stored in a local database on the mobile device and is periodically synch ...
```
Offline-first is no longer a requirement for native mobile apps. In fact, we encourage starting with an online-first native mobile app (the default synchronization mode in Mendix 11 is Online).
@RyanMocke Can you update the introduction to reflect this? It might even be worth changing the QuickStart to build an online app and focus more on the native aspects than offline-first development.

jdiehl commented on content/en/docs/quickstarts/native-mobile-app.md:
```diff
@@ -1,205 +1,176 @@
+* [Offline-first development](https://docs.mendix.com/refguide/mobile/building-efficient-mobile-apps/offlinefirst-data/): Native mobile apps run on devices that cannot guarantee a stable connection to the internet all the time. When developing these types of apps, it's important to design your app with this in mind. Data is stored in a local database on the mobile device and is periodically synch ...
 
-Before starting this tutorial, make sure you have completed the following prerequisites:
+* [Data synchronization](https://docs.mendix.com/refguide/mobile/building-efficient-mobile-apps/offlinefirst-data/synchronization/): In order to send and receive updates to the server, the native mobile application will need to synchronize its local database with the cloud environment’s server. This is triggered using the synchronize action in nanoflows and the synchronize to device action in mic ...
```
Same as above - please make sure to indicate this is optional

jdiehl commented on content/en/docs/quickstarts/native-mobile-app.md:
```diff
@@ -1,205 +1,176 @@
-    * The [Make it Native](/refguide/getting-the-make-it-native-app/) mobile app is available for Android and iOS devices
-    * Once installed, the app lets you quickly test your native mobile app as you develop it by connecting to your local development machine’s running copy of your app and displaying the app inside its mobile testing environment
-    * This lets you make changes to your app on your development machine, then instantly see those changes in the mobile app
+Before starting this tutorial, make sure you have completed the following prerequisites:
```
I would add one more prerequisite, to check that the the Mendix version you are using is compatible with Make It Native. You can link this page: /refguide/mobile/getting-started-with-mobile/prerequisites/#get-min-app

ConnorLand replied:
Done

jdiehl commented on content/en/docs/quickstarts/native-mobile-app.md:
```diff
@@ -1,205 +1,176 @@
+{{< figure src="/attachments/quickstarts/part2/7. Completed Nanoflow.png" width="450px" alt="Completed nanoflow">}}
 
-{{< figure src="/attachments/quickstarts/native-mobile-app/mobile-pic-2.png" width="150px" alt="Stegosaurus plant holder in a mobile device camera" class="no-border" >}}
+## 8. Deploying Your App
```
If you want to enable offline-first development, add a section to change the synchronization mode for the used entities from Online to All Objects

jdiehl reviewed (changes requested):
Thanks for updating the starter guide! Please see the inline comments about data synchronization not being enabled by default in mx11

Mocke44 committed:
Updated part 2 (Mobile quickstart) to reflect online first as defualt for MX 11

- Updated introduction to emphasize online-first as the default in Mendix 11
- Reframed offline-first and data synchronization as optional features
- Added prerequisite to check Mendix version compatibility with Make It Native
- Clarified that synchronize action in nanoflow is only needed for offline-first apps
- Added note in deployment section about switching entity sync mode for offline-first

ConnorLand committed:
Merge branch 'development' into QSG2025

ConnorLand committed:
Address comments

ConnorLand committed:
Formatting fixes

Mocke44 committed:
Updates to accomodate online first development as the current standard

Made several changes to highlight the fact that online first development is now the default behaviour unless otherwise configured

Mocke44 committed:
Merge branch 'QSG2025' of https://github.com/RyanMocke/docs into QSG2025

Mocke44 committed:
Added changes to address Jonathans Comments

Addressed that Online first development is available from mx 11

ConnorLand committed:
Remove duplication

ConnorLand committed:
Remove extra link pathways

ConnorLand committed:
Format

ConnorLand committed:
format

ConnorLand committed:
Fix links

ConnorLand committed:
Fix links and change header
